test/BlockManager.js

```diff
@@ -413,6 +438,19 @@ describe('BlockManager', function () {
 
       await voteManager.connect(signers[19]).commit(epoch, commitment1);
 
+      const votes2 = [1000, 2000, 3000, 4000, 5000, 6000, 7000, 8000, 9000];
+      const tree2 = merkle('keccak256').sync(votes2);
+
+      const root2 = tree2.root();
+      const commitment2 = utils.solidityKeccak256(
```

Comment:
Why 2 stakers are needed in this testcase, Here if I understand both 8 and 19 are participating, is it needed ?

Reply:
So i hvae only updated the previous test case in which 19th staker was partcipating earlier as it also has to propose a block in nest testCase. The new 8th staker is participating for commit and reveal state only

Reply:
oh okay 
can we move this 19th staker part to the next test case then, as that's where it's needed
Ideally, a test case should have only the things concerned with that testcase.
So when we read it it's clear.

Reply:
Ok cool, will do that.